src/lava/magma/compiler/compiler.py

```diff
@@ -251,6 +254,35 @@ def _group_proc_by_model(proc_map: PROC_MAP) \
 
         return grouped_models
 
+    # TODO: (PP) This currently only works for PyPorts - needs general solution
+    # TODO: (PP) Currently does not support 1:many/many:1 connections
```

Comment:
Issue or fix right away?
If too much work, do later but be sure we throw error messages as soon as possible if someone tries to create non-1:1 RefPort connections. We don't want this to fail deep in the compiler but right when the user does it.

Reply:
Issue #55 filed